src/FlashMessageMiddleware.php

```diff
@@ -22,7 +22,10 @@ class FlashMessageMiddleware implements MiddlewareInterface
     /** @var string */
     private $attributeKey;
 
-    /** @var callable */
+    /**
+     * @var callable
+     * @psalm-var callable(SessionInterface, string):array | array{ 0: class-string, 1: string }
```

Comment:
This union type seems borked - would be better to add a suppression on the assignment, than to mis-document this 🤔

Reply:
I struggled with this. For some reason, psalm [doesn't like](https://github.com/vimeo/psalm/issues/6230) string literals in the second element of an array-style callable. I've removed the union and sacrificed a chicken via an explicit @psalm-var in the method body.

Reply:
I've further improved this after @orklah helped me realize I was testing callability of the factory in a roundabout way.